packages/components/pages/f-mfa/src/components/_tests/Mfa.test.js

```diff
@@ -80,36 +106,195 @@ describe('Mfa', () => {
     });
 
     it.each([
-        ['show', true],
-        ['NOT show', false]
-    ])('should %s error message when hasSubmitError is %s', async (_, hasSubmitError) => {
-        // Arrange
+        ['enable', 'populated', 'abc123', undefined],
+        ['disable', 'unpopulated', '', 'true']
+    ])('should %s the submit button if the otp field is %s', async (_, __, optValue, expected) => {
+        // Act
         wrapper = await mountSut();
+        await wrapper.setData({ otp: optValue });
```

Comment:
Nit: typo in variable name, and it can also be simplified slightly
```suggestion
    ])('should %s the submit button if the otp field is %s', async (_, __, otp, expected) => {
        // Arrange
        wrapper = await mountSut();
        
        // Act
        await wrapper.setData({ otp });
```